docs/source/gammasim-tools-configuration.rst

```diff
@@ -0,0 +1,41 @@
+.. _Configuration:
+
+Configuration
+-------------
+
+Applications in gammasim-tools can be configured by the following four approaches, which are all equivalent:
```

Comment:
I consciously left it out, as I don't think it would be good practice to define a parameter several times with different values. I honestly think it is best to remove that completely and always raise an exception for duplicated definitions.

Reply:
I see. I was only worried in case the user doesnt realize he/she is defining it twice. It is very easy to forget about pre-defined environmental variables. I would prefer to keep the hierarchy but Im open to further discuss it.

Reply:
OK - this is a discussion outside of this pull request and I think until we have resolved it, we should not mention it. 

(I usual think it is better to force users to have a good setup than having magic happening in the background)